common/utils/formatErrorSummary.js

```diff
@@ -0,0 +1,9 @@
+const formatErrorSummary = errors => {
+  const errorSummary = []
+  errors.map(err => {
```

Comment:
The function discards the new array created by the map?
I think this should be a reduce:
```
const formatErrorSummary = errors => {
  return errors.reduce((errorSummary, error) => {
    return errorSummary.push({ text: err.msg, href: `#${err.param}-error` }
  }, [])
}

module.exports = { formatErrorSummary }
```